docs/getting-started/deployment/infrastructure-as-code/overview.md

```diff
@@ -0,0 +1,21 @@
+# Infrastructure as Code
+
+CarbyneStack has adopted Infrastructure as Code (IaC) as a core principle. IaC is the process of managing and provisioning infrastructure through code instead of manually deploying resources via kubectl, helm, etc. This allows for the infrastructure to be versioned, tested, and deployed in a repeatable manner.
+
+## Terraform
+
+CarbyneStack uses [Terraform by Hashicorp](https://www.terraform.io/) to provision infrastructure and deploy CarbyneStack on top of it.  Terraform has a large community and is widely used in the industry, with support for most major cloud providers through a large set of [providers](https://registry.terraform.io/browse/providers?product_intent=terraform).
+
+## Cloud Development Kit for Terraform (CDKTF)
+
+Terraform uses a custom syntax for their configuration language called HCL (HashiCorp Configuration Language).  While this is great for simple use cases, it can become difficult to manage as the complexity of the infrastructure grows.  To address this, CarbyneStack uses [CDKTF](https://learn.hashicorp.com/tutorials/terraform/cdktf), which allows you to use a common programming language, in CarbyneStack's case, Typescript, to define the infrastructure. This allows you to leverage the full power of a programming language to define your infrastructure, including the ability to use loops, conditionals, functions, and abstractions.  
```

Comment:
Extra whitespace `grows.   To`; please check whole PR regarding this "pattern"

Reply:
fixed